### PR TITLE
added jboss explanation (RK-2599)

### PR DESCRIPTION
**Pull request by BenTheKlein**

### CODE REVIEW

File: java-jboss/README.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+    <global-modules>
+        <module name="org.jboss.rookout" slot="main"/>
+    </global-modules>
+</subsystem>
+```
+
+Due to JavaEE class isolation we need to add the rookout sdk as a global module. 
+
+
+3. Extract the rookout.zip in the `<JBOSS_FOLDER>\modules\org\jboss` folder.
```
Comment (LiranLast) [inline java-jboss/README.md]: What's rookout.zip? Why do we need it?

File: java-jboss/README.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+
+Before following this guide we recommend reading the basic [Java + Rookout] guide.
+
+## Rookout Integration Explained
+
+There are 4 simple steps to integrate Rookout into your existing JBoss web application:
+
+1. Add the following JAVA_OPTS arguments in the standalone config script:
+```bash
+-javaagent:<PATH_TO_ROOK> -Djava.util.logging.manager=org.jboss.logmanager.LogManager -Djboss.modules.system.pkgs=org.jboss.byteman,org.jboss.logmanager
```
Comment (LiranLast) [inline java-jboss/README.md]: Why do we need so many flags?

Comment (BenTheKlein) [reply java-jboss/README.md]: We need that to fix the logger issues and class loading failures on startup

Comment (BenTheKlein) [reply java-jboss/README.md]: removing any of these flags will crash the program

File: java-jboss/README.md (hunk 1 above)
Comment (BenTheKlein) [reply java-jboss/README.md]: @LiranLast theoretically i can drop it at rook start; WDYT?

Comment (BenTheKlein) [reply java-jboss/README.md]: lets try to drop the rook.jar there in the first place

**Pull request merged**